src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const newState = { ...state };
+  const resultStates = [];
+
+  for (let i = 0; i < actions.length; i++) {
+    if (actions[i].type === `addProperties`) {
+      Object.assign(newState, actions[i].extraData);
+    }
+
+    if (actions[i].type === `clear`) {
```

Comment:
```suggestion
    if (actions[i].type === 'clear') {
```

Maybe better to use single quotes here and up.